wp-includes/widgets/class-wp-widget-image.php

```diff
@@ -42,17 +42,18 @@ public function __construct() {
 	 * @return void
 	 */
 	public function render_media( $attachment, $widget_id, $instance ) {
+		$instance    = wp_parse_args( $instance, array( 'size' => 'thumbnail' ) );
 		$has_caption = ! empty( $attachment->post_excerpt );
 
 		$image_attributes = array(
 			'data-id' => $widget_id,
 			'title'   => $attachment->post_title,
 			'class'   => 'image wp-image-' . $attachment->ID,
-			'style'   => 'width: 100%; height: auto;',
+			'style'   => 'max-width: 100%; height: auto;',
 		);
 
 		if ( ! $has_caption ) {
-			$image_attributes['class'] .= ' align' . $instance['align'];
+			$image_attributes['class'] .= ' ' . $instance['align'];
```

Comment:
So I think this change should be reverted, so we continue to prefix the alignment property with `align` only here when output as the prefixing of `align` is really only relevant in the context of a CSS classname.